core/field_label.js

```diff
@@ -100,4 +120,16 @@ Blockly.FieldLabel.prototype.doClassValidation_ = function(opt_newValue) {
   return String(opt_newValue);
 };
 
+/**
+ * Set the css class applied to the field's textElement_.
+ * @param {?string} cssClass The new css class name, or null to remove.
+ */
+Blockly.FieldLabel.prototype.setClass = function(cssClass) {
+  Blockly.utils.dom.removeClass(this.textElement_, this.class_);
```

Comment:
``removeClass`` expects an non-null element. this.textElement_ can be null if this field isn't initialized.

Reply:
Fixed and added test.